examples/connectivity/plot_adhd_covariance.py

```diff
@@ -26,7 +26,7 @@ def plot_matrices(cov, prec, title):
 
     # Put zeros on the diagonal, for graph clarity.
     size = prec.shape[0]
-    prec[range(size), range(size)] = 0
+    prec[list(range(size)), list(range(size))] = 0
```

Comment:
Just looked, turns out there's a `np.fill_diagonal` function.  Both it, and `diag_indices` are new in numpy 1.4, and nilearn targets >= 1.3.